src/main/java/de/cas/mse/exercise/EurekaNumbers.java

```diff
@@ -0,0 +1,50 @@
+package de.cas.mse.exercise;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class EurekaNumbers {
+
+	public List<Long> getEurekaNumbers(long startNumber, long endNumber) {
```

Comment:
you have side effects in the method since we are calculating them first and it is not a "get"-a-field